jvector-multirelease/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.github.jbellis</groupId>
+        <artifactId>jvector-parent</artifactId>
+        <version>0.9-SNAPSHOT</version>
+    </parent>
+    <artifactId>jvector</artifactId>
+    <name>JVector multirelease jar</name>
+    <properties>
+        <maven.install.skip>false</maven.install.skip>
```

Comment:
Nit: This is the default, do we need it as an explicit property? With the more obscure plugins, im +1 to be explicit about key settings, but can probably remove this.

Reply:
That makes sense. LGTM!